refinery/widgets/explorer_hierarchy_tree.py

```diff
@@ -470,16 +474,17 @@ def add_tag_index_ref(self, parent_dir_parts, tag_path, tag_index_ref):
                 parent_iid, 'end', iid=str(tag_id),
                 tags=("item", ), text=tag_path,
                 values=(cls1, cls2, cls3,
-                        tag_index_ref.meta_offset, pointer, tag_id))
+                        meta_offset, pointer, '%04X' % tag_id))
             self.tree_id_to_index_ref[tag_id] = tag_index_ref
         except Exception:
             print(format_exc())
 
     def add_folder_path(self, dir_parts):
         abs_dir_path = str(PureWindowsPath(*dir_parts))
         if abs_dir_path:
-            abs_dir_path += "\\"  # directories must end with a backslash.
-            #                       this is how we distinguish dirs from files
+            # directories must end with a backslash.
+            # this is how we distinguish dirs from files inside of maps.
```

Comment:
This documentation should really be on `_ensure_backslash_for_folder` so people know when and why to use that function.